test/functional/environments_controller_test.rb

```diff
@@ -24,7 +24,7 @@ class EnvironmentsControllerTest < ActionController::TestCase
     environment = Environment.new :name => "some_environment"
     assert environment.save!
 
-    get :edit, {:id => environment.to_param}, set_session_user
+    get :edit, {:id => environment.name}, set_session_user
```

Comment:
Used to be this way prior to https://github.com/theforeman/foreman/commit/8387eb79d86947a63b3856c5ce64936f39c2a9a0